scripts/startNewSchoolYear.js

```diff
@@ -40,7 +44,7 @@ appPromise
 			.updateMany(
 				{
 					federalState: { $in: federalStateIds },
-					ldapSchoolIdentifier: { $exists: false },
+					$or: [{ ldapSchoolIdentifier: { $exists: false } }, { ldapSchoolIdentifier: '' }],
```

Comment:
we could also add here currentSchoolYear condition 
`currentYear: currentSchoolYearId._id` instead of (or in addition to) `currentYear: { $exists: true }`

Reply:
Good point to do it similarly to the above situation for LDAP schools.
Why this is different here was my idea to basically adjust every school so no "remainers" are left behind.

For LDAP I wanted it to be more specific because there is more stuff that could go wrong.